conda/exports.py

```diff
@@ -80,7 +80,11 @@
 from conda.base.context import get_prefix as context_get_prefix, non_x86_linux_machines  # NOQA
 non_x86_linux_machines = non_x86_linux_machines
 
-from conda.base.constants import DEFAULT_CHANNELS  # NOQA
+from conda.base.constants import DEFAULT_CHANNELS       # NOQA
+from conda.models.package_info import NodeType          # NOQA
+from conda._vendor.auxlib.entity import EntityEncoder   # NOQA
```

Comment:
We should be preferring relative imports here.  It's probably not done in the whole file.  No better time to start though.